### PR TITLE
Validation: descriptions

### PR DESCRIPTION
**Pull request by SueADarby**

### CODE REVIEW

File: dynamics-nav/CAPTION-Function--RecordRef-.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "CAPTION Function (RecordRef)"
+description: Details the syntax for the CAPTION function (RecordRef) and gets the caption of the table that is currently selected.
```
Comment (dereklh7) [inline dynamics-nav/CAPTION-Function--RecordRef-.md:3]: "gets" the caption? Maybe "provides" would be a better word?

File: dynamics-nav/CAPTION-Function--TestPage-.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "CAPTION Function (TestPage)"
+description: Details the syntax for the CAPTION function (TestPage). This property gets the caption of the test page.
```
Comment (dereklh7) [inline dynamics-nav/CAPTION-Function--TestPage-.md:3]: "gets" the caption? Maybe "provides" would be a better word?

File: dynamics-nav/CLOSE-Function--RecordRef-.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 ---
 title: "CLOSE Function (RecordRef)"
+description: Details the syntax for the CLOSE Function (RecordRef). The RecordRef of a record in the table that you want to close.
+
```
Comment (dereklh7) [inline dynamics-nav/CLOSE-Function--RecordRef-.md:4]: Delete Lines 4 and 5. Don't have blank lines in metadata.

**Comment by SueADarby** [comment]
**Global effort to fix build validation errors**
@buck1ey 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by edupont04 — APPROVED** [review]

**Pull request merged**